ajax/libs/email-autocomplete/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "email-autocomplete",
+  "filename": "jquery.email-autocomplete.min.js",
+  "description": "jQuery plugin that displays in-place autocomplete suggestions for email input fields.",
+  "keywords": [
+    "jquery-plugin",
+    "ecosystem:jquery",
+    "autocomplete",
+    "suggestion",
+    "typeahead",
+    "domain",
+    "email",
+    "form"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/10w042/email-autocomplete/"
+  },
+  "license": "MIT",
+  "author": "Low Yong Zhen <yz@stargate.io>",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/10w042/email-autocomplete.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*.js"
```

Comment:
Let's modify from `"**/*.js"` to `"**/*"`, because maybe this library add `.css` files under `dist` in future version.
And we usually use `**/*` which "basePath" is "dist".
Thank you.